wiki/Announcements/osu!talk/en.md

```diff
@@ -1,90 +1,93 @@
-osu!talk
-===============
+# osu!talk
```

Comment:
:( I think I was going to shorten the underline to
```
osu!talk
========
```
but then thought, "Ah, heck with it, if I'm going to fix the underline, I might as well just change styles."